Patch by Kamil Dudka <kdudka@redhat.com> for libunicap >= 0.9.12, to get it building on Red Hat
Enterprise Linux 5 (or derivates); see https://bugzilla.redhat.com/show_bug.cgi?id=635377#c2 for
more details.

--- libunicap-0.9.12/cpi/v4l2cpi/buffermanager.c		2010-08-17 18:13:08.000000000 +0200
+++ libunicap-0.9.12/cpi/v4l2cpi/buffermanager.c.includes	2010-10-04 21:44:09.942065713 +0200
@@ -1,5 +1,8 @@
 #include "config.h"
 
+#include <sys/time.h>
+
+#include <linux/ioctl.h>
 #include <linux/types.h>
 #include <linux/videodev2.h>
 #include <semaphore.h>
